projects/kit/components/line-clamp/line-clamp.component.ts

```diff
@@ -43,11 +57,22 @@ export class TuiLineClampComponent implements AfterViewInit {
     @tuiDefaultProp()
     content: PolymorpheusContent = '';
 
+    @Input()
+    @tuiDefaultProp()
+    showHint = this.options.showHint;
```

Comment:
Do we really need `@Input()`?

May be static DI-configuration is enough ?

Reply:
I'm not sure about this, in my case using DI would be enough, so I can remove the input

Reply:
Let's remove it for now.

We will add it later if if there is a need for it.

Reply:
Done